pallets/runtime/src/general_tm.rs

```diff
@@ -208,22 +249,118 @@ decl_module! {
         }
 
         /// It resumes the verification of rules for `ticker` during transfers.
+        ///
+        /// # Arguments
+        /// * origin - Signer of the dispatchable. It should be the owner of the ticker
+        /// * ticker - Symbol of the asset
         pub fn resume_asset_rules(origin, ticker: Ticker) -> DispatchResult {
             Self::pause_resume_rules(origin, ticker, false)?;
 
             Self::deposit_event(Event::ResumeAssetRules(ticker));
             Ok(())
         }
+
+        /// To add/remove the default trusted claim issuer for a given asset
+        /// Addition - When the given element is not exist
+        /// Removal - When the given element is already present
+        ///
+        /// # Arguments
+        /// * origin - Signer of the dispatchable. It should be the owner of the ticker.
+        /// * ticker - Symbol of the asset.
+        /// * trusted_issuer - IdentityId of the trusted claim issuer.
+        fn modify_default_trusted_claim_issuer(origin, ticker: Ticker, trusted_issuer: IdentityId) -> DispatchResult {
+            let sender_key = AccountKey::try_from(ensure_signed(origin)?.encode())?;
```

Comment:
Was there a reason to combine adding / removing in a single function? Seems like it might be confusing.

Reply:
The only reason to add this way is to avoid having duplicate code. Only one line difference in add & remove. If you guys think that is confusing then I can convert it into 2.